libr/core/cmd_open.c

```diff
@@ -33,7 +33,7 @@ static const char *help_msg_o[] = {
 	"oo","+","reopen current file in read-write",
 	"ood","[r] [args]","reopen in debugger mode (with args)",
 	"oo[bnm]"," [...]","see oo? for help",
-	"op"," [fd]", "prioritize given fd (see also ob)",
+	"op"," [fd]", "select the given fd as current file (see also ob)",
```

Comment:
maybe "select the specified file descriptor over the others". but that's what "priorize" says to me

Reply:
Selection is the act of choosing one (or more) choices over another where as prioritization is typically used for ordering (eg. priority queue; where there are multiple items in an order). Using "select the specified" would make the text "over the others" redundant. I think the confusing part about prioritize is that there can only be one selected fd at a time from my understanding. There is no hierarchy after the selected fd; they are all equal priority.

Reply:
"select" makes more sense. A descriptor is either the current one `io->desc` or not. There is no ordering involved.